localDependencies.cmake

```diff
@@ -0,0 +1,6 @@
+# this file set variables pointing to local installation of dependencies.
```

Comment:
I don't think this file is needed, it's part of the cmake usage, and depends on the user setup.
We may state that in the documentation, but don't need to include it in Radium sources.

Reply:
I agree.
For now, it is here just for the discussion and some tests, it is easier to have just 1 parameter to the cmake command line than several ones (and less error prone)). It will be removed when the PR wil be in final state.

Reply:
A proper way to that with modern cmake is to use Toolchain files, which contains all the configuration option that you want to locally set to cmake. If we want to push this approach, we may have a default toolchain file which uses dependencies loaded dynamically by cmake, and maybe one or two examples of libraries on standard platforms (e.g., vanilla ubuntu).

Reply:
I'm not sure that TOOL_CHAIN_FILE is the proper way to do that. It is intended for cross-compilation if I understand well the doc : 
> This variable is specified on the command line when cross-compiling with CMake.

Reply:
Well, it's up to the user to define how he wants the different options, with tool chaine, with -C FILE, or directly with -D.
So I propose : we do not say anyting about that in the doc, just document the options.
I have finished on Core, create a script that tests the different combinations of local installed vs fetch... seems to work (but the script is tailored on my own install, not for sharing).